plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -1667,12 +1690,25 @@ bool producer_plugin_impl::remove_expired_trxs( const fc::time_point& deadline )
                } else {
                   fc_dlog(_trx_failed_trace_log, "[TRX_TRACE] Speculative execution is EXPIRING PERSISTED tx: ${txid}", ("txid", txid));
                }
+
+               if (_trx_trace_dump_log.is_enabled(fc::log_level::debug)) {
+                   // Dump transaction
+                   auto entire_trx = chain_plug->get_entire_trx(this->_unapplied_transactions.get_trx(txid)->packed_trx()->get_transaction());
+                   fc_dlog(_trx_trace_dump_log, "[TRX_TRACE] ${entire_trx}", ("entire_trx", entire_trx));
```

Comment:
Add "EXPIRING PERSISTED tx:" to log output.

Reply:
Fixed.